Project4/index.html

```diff
@@ -0,0 +1,86 @@
+        </svg></a>
+        <a href="#cart"><svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36"><defs><style>.cls-1,.cls-2,.cls-3{fill:none;stroke:#231f20;stroke-miterlimit:10;}.cls-2{stroke-width:0.97px;}.cls-3{stroke-width:1.1px;}.cls-4{fill:#231f20;}</style></defs><title>GD2EventIcons</title><path class="cls-1" d="M11.93,25.48c.45,1.37.89,2.49,1.22,3.27,2,.24,3.66.37,5,.46a56.41,56.41,0,0,0,7.53.15c1.76-.11,2.05-.24,3.5-.23,2,0,3.46.28,3.57-.08s-.46-.58-.6-.68c-2.18-1.45-5.67-.87-5.86-.84a34,34,0,0,1-11.19-.38,2.57,2.57,0,0,1-1.55-2.28,2.54,2.54,0,0,1,.75-1.71,53.4,53.4,0,0,0,9.17,0,52.9,52.9,0,0,0,8.93-1.51,20.71,20.71,0,0,0,.72-2.26,23.62,23.62,0,0,0,.4-9.69,84.71,84.71,0,0,1-11.65.83A85.62,85.62,0,0,1,11,9.92c-.23-.21-3.34-3-4.7-2.1-.51.34-.79,1.22-.53,1.53s1.13-.25,2.05.07a2.59,2.59,0,0,1,1.37,1.42,107.53,107.53,0,0,0,2,11.94C11.53,24.22,11.93,25.48,11.93,25.48Z"/><circle class="cls-1" cx="29.86" cy="32.32" r="1.86"/><circle class="cls-1" cx="15.28" cy="32.32" r="1.86"/><path class="cls-1" d="M21.18,5l4.6,4.21q-1.74.06-3.63.06c-2,0-4-.07-5.76-.16l-2-1.8Z"/><path class="cls-2" d="M12,5.12l1.35-.52a4.32,4.32,0,0,0,3.12.28,1.29,1.29,0,0,0,.69-1.76l1.52-.59L21.32,5l-6.8,2.37Z"/><polygon class="cls-3" points="1.67 28.11 7.15 29.84 9.18 19.97 3.71 18.25 1.67 28.11"/><path class="cls-3" d="M4.53,14.26l1,.31a2.2,2.2,0,0,0,1.36,2A1.7,1.7,0,0,0,9,15.68l1,.3-.75,3.65L3.78,17.9Z"/><path class="cls-3" d="M6.32,33.83l-1-.32a2.2,2.2,0,0,0-1.37-2,1.71,1.71,0,0,0-2.15.9l-1-.31.75-3.64,5.48,1.72Z"/><path class="cls-4" d="M4.2,27.94l0-.15-1.08-.65,0,.19-.25-.06.19-.94.25.05,0,.18,1.25-.19,0-.14.25.05-.13.65-.25-.05,0-.2-.3,0L4,27.35l.25.15,0-.21.25.05L4.46,28Zm-.29-1.19-.71.1.61.37Z"/><path class="cls-4" d="M4.57,26.1l0-.19-1.16-.23,0,.19-.26-.05.13-.64c.12-.61.51-.9,1-.8s.79.51.66,1.17l-.12.6Zm.13-.65a.58.58,0,0,0-.46-.77c-.37-.08-.62.14-.71.56l0,.15,1.17.23Z"/><path class="cls-4" d="M4.93,24.32l0-.2-1.16-.28,0,.21L3.51,24l.09-.46,1-.33-.83-.7.09-.46.26.05,0,.21,1.17.18,0-.2.26.05L5.45,23,5.19,23l0-.2-.87-.14.75.65v0l-.94.32.86.2,0-.2.26.05-.14.69Z"/><path class="cls-4" d="M5.37,22.11l0-.2-1.17-.24,0,.21L4,21.83l.14-.7.25.05,0,.21,1.17.23,0-.21.25.05-.13.7Z"/><path class="cls-4" d="M5.61,20.88l0-.2-1.17-.24-.06.31.27.06-.06.27L4.12,21l.29-1.46.51.1,0,.28-.27-.06-.06.31,1.17.24,0-.21.26.05-.14.71Z"/><path class="cls-4" d="M6.14,26.73a.88.88,0,1,1,1-.68A.85.85,0,0,1,6.14,26.73Zm.29-1.42a.57.57,0,1,0-.23,1.12A.59.59,0,0,0,6.92,26,.57.57,0,0,0,6.43,25.31Z"/><path class="cls-4" d="M6.3,24.26l.9.18,0-.21.25.05L7.37,25l-.26-.05,0-.19L6,24.49l0,.19-.26-.05.08-.44L7,23.57l-.79-.15,0,.2-.26-.05L6,22.9l.26.05,0,.18,1.43.29,0,.16Z"/><path class="cls-4" d="M7.49,23l0-.19-1.17-.23,0,.19-.25-.05.25-1.25.48.09-.06.28-.22-.05-.1.5.45.09.1-.51.24.05-.1.51.48.09.1-.5-.23,0,.06-.28.48.1-.25,1.26Z"/>
+        </svg></a>
+        </div>
+        <img class="thumbnail" src="FalveyHall.jpg" alt="BrownCenter"></b>
```

Comment:
Stray `b` element here:
```suggestion
        <img class="thumbnail" src="FalveyHall.jpg" alt="BrownCenter">
```